lib/texts.js

```diff
@@ -153,6 +153,11 @@ module.exports = {
 	location: () => 'Getting current location',
 	locationCheck: template`Checking if current location ${0} "${1}"`,
 
+	takeScreenshot: () => 'Take screenshot',
```

Comment:
Other messages here are written in present continues tense. So, should probably be "Taking screenshot"